### ISSUE TITLE
[scallop] Error: Validation failure for 'from-chapter' option parameters: 

### ISSUE THREAD

**Issue by mstevens** [Aivean/royalroad-downloader#13]
Hi,

Trying to download, but I get a validation error every time:

```
mstevens@osaka:~/royal-road-download % java -jar royalroadl-downloader-assembly-1.3.0.jar https://www.royalroadl.com/fiction/18619
[scallop] Error: Validation failure for 'from-chapter' option parameters: 
```
My java is:

```
mstevens@osaka:~/royal-road-download % java -version
java version "1.8.0_77"
Java(TM) SE Runtime Environment (build 1.8.0_77-b03)
Java HotSpot(TM) 64-Bit Server VM (build 25.77-b03, mixed mode)
```

**Comment by gibbed** [comment]
It's a bug with the recent version, I found you can work around the problem by adding `--from-chapter=1`.

**Comment by Aivean** [comment]
Well, what can I say, shame on me for the lack of testing. Fixed in 1.3.1.

Thank you for reporting!

**Comment by mstevens** [comment]
Just to confirm I tested and it works nicely now. Thanks!
